packages/flame/README.md

```diff
@@ -1 +1 @@
-../../README.md
\ No newline at end of file
+../../README.md
```

Comment:
This file shouldn't have any changes in it, I'm guessing it is a different type of line break? :thinking:

Reply:
Yeah, I reverted it, it was failing the CI. `markdown-check` added it for some reason

Reply:
Ah, it might depend on what operating system you run `markdown-fix` on what the results will be